doc/source/tutorial/stats.rst

```diff
@@ -1190,3 +1191,246 @@ because the p-value is very low and the MGC test statistic is relatively high.
 The MGC-map indicates a **strongly nonlinear relationship**. The optimal scale
 in this case is **equivalent to the local scale**, marked by a red spot on the
 map.
+
+Quasi-Monte Carlo
+-----------------
+
+Before talking about Quasi-Monte Carlo (QMC), a quick introduction about Monte
+Carlo (MC). The MC method was called in statistics 'model sampling'. It is
+a method which aims at trying out hypothesis. Once all the hypothesis we
+have defined have been tested, we can compute some statistics such as the
+mean of a given quantity of interest. A scenarios is defined by some variables.
+So, in order to map all scenarios, we use a random number generator.
+
+Generating random numbers is a more complex problem than it sounds. Simple
```

Comment:
I could remove this sentence. But also, it's not that trivial if you consider all the literature about simple random number generation. PRNG, TRNG, quantum things, all the different algorithms, all the statistical tests involved to validate the generators.

I can also propose to add an adjective before random number, such as good, uniform. I believe that reading the rest of the text clarifies all of this.

Reply:
Maybe just insert "with specific properties" after "random numbers"? I don't mind the sentence, it's indeed nontrivial - although that's for the implementers of the methods; to the users it's a matter of a couple of lines of code using `numpy.random`.

Reply:
I mainly wanted to emphasize that QMC can have certain benefits compared to using uniform RNGs, no matter whether generating uniform rvs is difficult or not. So I would try to emphasize the following points:

- classical MC usually guarantees a rate of convergence of N**(-1/2)
- one disadvantage of using pseudo random numbers is that they exhibit clustering behaviour, i.e. they do not evenly cover the space [0,1]^d
- QMC tries to overcome this by relying on deterministic sequences

you do mention all these points in the rest of the tutorial, I just find the introduction misleading, maybe it can be restructured slightly to give a better motivation for using QMC?

Reply:
I reworked the introduction as you suggested with the entry from wikipedia. I am not sure about listing all the properties right away though. I prefer to walk the user through it instead. IMO, this might be too overwhelming for someone reading about MC for the first time otherwise.